src/Symfony/Bridge/Twig/Command/LintCommand.php

```diff
@@ -226,7 +226,7 @@ private function displayJson(OutputInterface $output, array $filesInfo)
         return min($errors, 1);
     }
 
-    private function renderException(OutputInterface $output, string $template, Error $exception, string $file = null)
+    private function renderException(SymfonyStyle $output, string $template, Error $exception, string $file = null)
```

Comment:
This is a minor. We pass a `SymfonyStyle` and we use the `$object` as a `SymfonyStyle`.